db/seeds.rb

```diff
@@ -34,225 +106,207 @@
 #{'╚═╝░░░░░░╚════╝░░╚════╝░╚═════╝░'.magenta}  #{'░░░░░╚═╝'.magenta}  ╚═╝░░╚═╝╚══════╝╚══════╝"
 puts "Donating the food and resetting the database".red.blink
 puts ''
-# With proper dependencies, deleting the users should delete the entire db.
+# Due to model associations, this will delete msot of the database.
 User.destroy_all
 
-#############################################################################
-#----------------------------DEMO PERSONA-----------------------------------#
-#############################################################################
-
-# Demo user Justin ('receiver' then time passes and he becomes a 'giver').
-justin = User.create!(
-  email: 'justin@foodfor.all',
-  username: 'Justin',
-  password: '123456',
-  address: '5333 Av. Casgrain'
-)
-puts "#{'✓'.light_green} Demo persona: #{justin.username.light_cyan} has been created."
-puts '--------------------------------------------------------------------'.light_black
-
 #############################################################################
 #-------------------------SEED DB WITH ALLERGENS----------------------------#
 #############################################################################
+puts '----------------------------Allergens-------------------------------'.light_black
 
 allergen_counter = 0
 @allergens_list.length.times do
+  next if Allergen.count == 8
```

Comment:
This won't delete and re seed the Allergens if they already exist (ex: running `rails db:seed`)